activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Specify callback in `has_secure_token`
+
+        ```ruby
```

Comment:
Can you remove the indentation on this code block? :bow:

Reply:
Done!